src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -535,7 +535,7 @@ private boolean areProxyCredentialsAvailable() {
             logger.debug("Proxy user and password set as System Environment Variables : {} and {}" , 
                     HTTP_PROXY_USER, HTTP_PROXY_PASSWORD);
         } else {
-            logger.debug("No proxy set - No settings found as System Environment Variable: {}. "
+            logger.warn("No proxy set - No settings found as System Environment Variable: {}. "
```

Comment:
having no proxy is not a warning. since this is no fixture method itself, I'd use trace level instead